http/wsclient.go

```diff
@@ -236,7 +236,7 @@ func (c *WSConn) write(msg []byte) error {
 	}
 
 	c.conn.SetWriteDeadline(time.Now().Add(writeWait))
-
+	c.conn.EnableWriteCompression(config.GetConfig().GetBool("agent.http.websocket.enable_write_compression"))
```

Comment:
this should be done in the connect function no ?

Reply:
agree, moved it